src/components/Content.tsx

```diff
@@ -2,7 +2,7 @@ import type { JSX, RenderableProps } from "preact";
 
 export function Content({ children }: RenderableProps<unknown>): JSX.Element {
   return (
-    <article class="prose prose-lg max-w-none bg-slate-200 px-10 md:px-20 lg:px-40 py-10 dark:prose-invert prose-headings:flex prose-headings:flex-row prose-headings:items-center dark:bg-slate-800">
+    <article class="prose prose-lg max-w-none flex flex-col place-items-center bg-slate-200 px-10 md:px-20 lg:px-40 py-10 dark:prose-invert prose-headings:flex prose-headings:flex-row prose-headings:items-center dark:bg-slate-800">
```

Comment:
Ugh. Just looked at the site again, and I very much regret this. We'll want to fix it later. Maybe as part of #28 if you want to do some CSS, maybe not.